packages/trimerge-sync/src/TrimergeClient.ts

```diff
@@ -647,6 +661,24 @@ export class TrimergeClient<
   }
 
   public shutdown(): Promise<void> | void {
```

Comment:
I wonder if we should just make this always return a promise for simplicity, since it doesn't need to be performant